Package@swift-5.11.swift

```diff
@@ -0,0 +1,160 @@
+// swift-tools-version: 5.11
+
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
```

Comment:
2024, now 🍾

Reply:
Yes but the manifest file is from 2023. 😛